pkg/specgenutil/volumes_test.go

```diff
@@ -15,39 +15,39 @@ func Test_validChownFlag(t *testing.T) {
 		{
 			name: "U true",
 			args: args{
-				flag: "U=true",
+				flag: "true",
 			},
 			want:    true,
 			wantErr: false,
 		},
 		{
 			name: "U true case does not matter",
```

Comment:
...and this needs the `U` removed...

Reply:
Renamed to `case-insensitive true` & added a `case-insensitive false` test case